java/adapter/avro/src/main/java/org/apache/arrow/AvroToArrowUtils.java

```diff
@@ -164,6 +182,33 @@ private static Consumer createConsumer(
     return consumer;
   }
 
+  private static Consumer createArrayConsumer(Schema schema, String name, BufferAllocator allocator) {
+
+    Consumer delegate = createConsumer(schema.getElementType(), schema.getElementType().getName(), allocator);
+    ListVector listVector = ListVector.empty(name, allocator);
+    listVector.setDataVector(delegate.getVector());
+
+    return new AvroArraysConsumer(listVector, delegate);
+  }
+
+  private static Consumer createMapConsumer(Schema schema, String name, BufferAllocator allocator) {
+
+    StructVector structVector =
+        new StructVector(name, allocator, FieldType.nullable(ArrowType.Struct.INSTANCE), null);
```

Comment:
comment.

Reply:
fixed.

Reply:
sorry I forget that github doesn't support highlighting code, I meant comment what the null value is.